src/table-doc.js

```diff
@@ -0,0 +1,53 @@
+import { escapeHTML } from './strings';
+
+export function tableDoc(code) {
+    try {
+        let json = JSON.parse(code);
+        // support language specific overrides
+        // if (json.languages && json.langauges[language]) {
+        //     Object.assign(json, json.languages[language]);
+        // }
+        
+        const td = [];
+
+        td.push(`<table>`);
+        td.push(`<tr><th>Name</th><th>Type</th></tr>`);
+        
+        if(json.columns){
+            td.push(tableHeaders(json.columns));
+        }
+        // if(json.example){
+        //     td.push(tableData(json.data));
+        // }
+
+        td.push(`</table>`);
+        console.log(td.join('\n'));
+        return td.join('\n');
+        
+    }
+    catch (ex) {
+        console.log(ex);
+        return '`failed to render %jsonblock: `' + ex.message + '`';
+    }
+}
+
+function tableData(json){
+    let data = [];
+    json.map(elem => '<tr>' + elem.map(a => `<td>${a}</td>`).join('') + '</tr>').join('\n');
+    return data.map((a) => a.join('')).join('\n');
```

Comment:
https://github.com/Codewars/marked-extensions/blob/612cd7d09890b8bd611bb85ac95562ae385ed55c/src/table-doc.js#L34-L38

`tableData` now returns empty string because nothing is added to `data`.

Return the result of `map`:
```javascript
function tableData(json){
  return json.map(elem => '<tr>' + elem.map(a => `<td>${a}</td>`).join('') + '</tr>').join('\n');
}
```
or
```javascript
const tableData = json =>
  json.map(elem => '<tr>' + elem.map(a => `<td>${a}</td>`).join('') + '</tr>').join('\n');
```